### PR TITLE
Remove public master host argument in integration tests

### PR DESCRIPTION
**Pull request by mellenburg**
## High Level Description
This parameter is essentially useless as it doesnt need to be a public address and in any circumstance where it is actually used, the same private IP is provided for both this and the MASTER_HOSTS argument

## Checklist for all PR's

  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [ ] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

### CODE REVIEW

File: test_util/dcos_api_session.py [hunk 1]
```diff
@@ -89,32 +87,27 @@ def api_request(self, method, path_extension, *, scheme=None, host=None, query=N
 
 
 class DcosApiSession(ARNodeApiClientMixin, RetryCommonHttpErrorsMixin, ApiClientSession):
-    def __init__(self, dcos_url: str, masters: list, public_masters: list,
-                 slaves: list, public_slaves: list, default_os_user: str,
-                 auth_user: Optional[DcosUser]):
+    def __init__(self, dcos_url: str, masters: list, slaves: list,
```
Comment (orsenthil) [inline test_util/dcos_api_session.py]: Opportunity for hanging indent.

File: test_util/test_upgrade_vpc.py [hunk 2]
```diff
@@ -603,9 +601,9 @@ def run_test(self) -> int:
 
 
 class DcosApiSessionFactory(VpcClusterUpgradeTestDcosApiSessionFactory):
-    def apply(self, dcos_url: str, masters: List[str], public_masters: List[str], slaves: List[str],
+    def apply(self, dcos_url: str, masters: List[str], slaves: List[str],
```
Comment (orsenthil) [inline test_util/test_upgrade_vpc.py]: opportunity to change indentation style.

**Comment by greggomann** [comment]
LGTM 🚀

**Comment by mellenburg** [comment]
This PR is pretty trivial and is a requirement for #1508 so it will just be merged there. Thus, this ticket is now closed

**Pull request closed without merging**